Modules/Microsoft365DSC/Modules/M365DSCDRGUtil.psm1

```diff
@@ -1160,7 +1272,7 @@ function Update-DeviceConfigurationPolicyAssignment
             $deviceManagementPolicyAssignments += @{'target' = $formattedTarget}
         }
         $body = @{'assignments' = $deviceManagementPolicyAssignments} | ConvertTo-Json -Depth 20
-        #write-verbose -Message $body
+        write-verbose -Message $body
```

Comment:
I think this is still here for troubleshooting and should be commented out. Correct?

Reply:
Removed debugging trace